anyway/backend_constants.py

```diff
@@ -108,6 +108,17 @@ class ResolutionCategories(Enum):
         ResolutionCategories.SUBURBAN_ROAD,
     ]
 
+    class Source(Enum):
+        YNET = "ynet"
+        WALLA = "walla"
+        TWITTER = "twitter"
+
+    SUPPORTED_SOURCES: List[Source] = [
+        Source.YNET,
+        Source.WALLA,
+        Source.TWITTER,
+    ]
```

Comment:
What is `SUPPORTED_SOURCES` list used for? It can be generated from `Source` class by `[e for e in Source]`

Reply:
True, but as I understand there might be some unsupported sources in the near future.
Plus it makes it more consistent with the code for resolutions.